roofit/xroofit/inc/RooFit/xRooFit/xRooBrowser.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
```

Comment:
Also we should use include guards here. But since this `xRooBrowser.h` will be the one header file that is shared by RooFit and your standalone `xRooFit`, indeed you have to do something to avoid collisions.

Maybe you can select the appropriate header guard with another `ifdef XROOFIT_NAMESPACE`?